backend/src/test/java/ai/verta/modeldb/HydratedServiceTest.java

```diff
@@ -1513,14 +1505,8 @@ public void findHydratedExperimentRunsTest() {
             .addPredicates(keyValueQueryLoss)
             .build();
 
-    try {
-      hydratedServiceBlockingStub.findHydratedExperimentRuns(findExperimentRuns);
-      fail();
-    } catch (StatusRuntimeException e) {
-      Status status = Status.fromThrowable(e);
-      LOGGER.warn("Error Code : " + status.getCode() + " Description : " + status.getDescription());
-      assertEquals(Status.PERMISSION_DENIED.getCode(), status.getCode());
-    }
+    response = hydratedServiceBlockingStub.findHydratedExperimentRuns(findExperimentRuns);
+    assertEquals("Expected response not found", 0, response.getHydratedExperimentRunsCount());
```

Comment:
Now we are checking permission first so if given project id is not valid it will return empty list instead of PERMISSION_DENIED